packages/math/src/Rect.ts

```
@@ -12,8 +15,20 @@
   /** @internal */
   _height: number;
   /** @internal */
-  _onValueChanged: () => void = null;
-
+  get _onValueChanged(): () => void {
+    if (this._onValueChangedCallback === emptyFunc) {
+      return null;
+    }
+    return this._onValueChangedCallback;
+  }
+  set _onValueChanged(callback: () => void | null) {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)